iotilecore/RELEASE.md

```diff
@@ -8,6 +8,12 @@ All major changes in each released version of `iotile-core` are listed here.
   This is necessary now that `iotile-emulate` no longer supported python 2 and
   requires asyncio inside its emulated tiles.
 
+## 3.26.6
```

Comment:
If you want to push a new release tag, you need to move all of the existing stuff in ##HEAD to that block also, since those commits will be included in the new version of the software.

Reply:
So my recommendation is that it's probably better move your changes back to HEAD for now, since we do have pending HEAD changes already that we might want to review before releasing a version.

Reply:
Thank you very much for your recommendations, unfortunately I was out of reach to my laptop and did not fulfill them. 
 Thank you one again for help.